main.py

```diff
@@ -268,15 +273,26 @@ def populate(ctx, skip_versions,
 
     git = instanciate_git_connector(configuration, git_factory_provider, tmp_dir, repo_dir)
 
+    if len(configuration.source_bugs) == 0:
+        logging.error("No synchro because parameter 'OTTM_SOURCE_BUGS' no defined")
```

Comment:
This has to be checked before, and should raise a ConfigurationValidationException
It should be also checked in the check() function